.github/workflows/test.yml

```
@@ -0,0 +1,48 @@
+      - v1.0
+  pull_request:
+    branches:
+      - v1.0
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)